sdk/appconfiguration/azure-spring-cloud-feature-management/README.md

```diff
@@ -259,6 +259,171 @@ Options are available to customize how targeting evaluation is performed across
     }
 ```
 
+## Dynamic Features
+
+Using `FeatureManager`'s `getVariantAsync` method allows for returning of dynamic features based off a feature evaluator. Usage:
+
+```java
+featureManager.getVariantAsync("DiscountBanner", DiscountBanner.class)
+```
+
+This example shows requesting a Dynamic Feature named "DiscountBanner", it requires the name of the Dynamic Feature and the type that needs to be returned.
+
+### Defining a Dynamic Feature
+
+A Dynamic Feature can be defined in the local configuration file using the following format.
+
+```yaml
+feature-management:
+  dynamic-features:
+    DiscountBanner:
+      assigner: "Microsoft.Targeting"
+      variants:
+        -
+          name: Big
+          default: true
+          configuration-reference: "DiscountBanner:Big"
+          assignment-parameters:
+            ...
+        -
+          name: Small
+          default: false
+          configuration-reference: "DiscountBanner:Small"
+          assignment-parameters:
+            ...
+feature-variant:
```

Comment:
Do you think `feature-variant` or `feature-variants` is better considering we'll be defining all the feature variants here.